test/Chainweb/Test/Utils.hs

```diff
@@ -1013,28 +1023,29 @@ node
     -> logger
     -> MVar (PeerInfo, Port)
     -> ChainwebConfiguration
-    -> Int
+    -> FilePath
+       -- ^ pact db dir
+    -> FilePath
+       -- ^ rocksdb dir
+    -> Word
         -- ^ Unique Node Id. The node id 0 is used for the bootstrap node
     -> IO ()
-node testLabel rdb rawLogger peerInfoVar conf nid = do
+node testLabel rdb rawLogger peerInfoVar conf pactDbDir rocksDbDir nid = do
```

Comment:
Idk if this is the best PR to do it, but I think `withChainweb` should not take these directories as an input. `ChainwebConfiguration` contains a database directory, and that should be the base directory for all databases. That further means that `node` stops needing to take them as input because it takes a `ChainwebConfiguration` as well.

Reply:
yeah, i don't think this PR should do it. we should make a ticket.